### DISCUSSION TITLE
Keyboard shortcuts for Ente Auth WebApp to: Search and Copy codes

### DISCUSSION THREAD

**Discussion by homeostashish** [ente-io/ente#3080] [Enhancements]
I would like to suggest Keyboard shortcuts on the webapp for:
1. Focusing on the search field.
2. Navigation (if multiple entries appear for the same search query)
3. Copy the current code
4. Copy the next code

Thank You.

**Comment by JoelAzaria** [comment]
How is this not a thing?   At the very least a shortcut for the search box is a must have.
Please.

Thanks.

**Comment by rockyroed** [comment]
Number 1 is essential to my use case. I'd really love having a shortcut for the search bar. The other 3 is also great, though. Hope these will be implemented.

**Reply by JoelAzaria** [reply]
Announced in #1254, ctrl+f shortcut key is now avail for function # 1


#'s 2, 3 and 4 still wanted!!
